content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,411 @@
+    "endedAt": "2024-03-29T00:59:22.975723884Z",
+    "error": null,
+    "estimatedRunningDurationMinutes": 0,
+    "generation": 8,
+    "groups": [
+        {
+            "id": "eb988b5e2dcb11ec840712b3b5263ef4",
+            "name": "Default Group"
+        }
+    ],
+    "healthStatus": "OK",
+    "id": "9825d5e8-5213-433b-8fbf-1945bb4496e1",
+    "logMessages": [
+        {
+            "message": "Task created",
+            "timestampAt": "2024-03-29T00:56:16.98131301Z"
+        },
+        {
+            "message": "Task is running",
+            "timestampAt": "2024-03-29T00:56:16.981315803Z"
+        },
+        {
+            "message": "Preparing parameters",
+            "timestampAt": "2024-03-29T00:56:18.622573215Z"
+        },
+        {
+            "message": "Creating datastore",
+            "timestampAt": "2024-03-29T00:56:19.797636611Z"
+        },
+        {
+            "message": "Validating datastore",
+            "timestampAt": "2024-03-29T00:56:36.120087916Z"
+        },
+        {
+            "message": "Applying protection policy",
+            "timestampAt": "2024-03-29T00:57:36.332756275Z"
+        },
+        {
+            "message": "Task succeeded",
+            "timestampAt": "2024-03-29T00:59:22.975739714Z"
+        }
+    ],
+    "name": "Provisioning datastore VVOL-ds1",
+    "parentTask": null,
+    "progressPercent": 100,
+    "recommendations": [],
+    "resourceUri": "/data-services/v1beta1/async-operations/9825d5e8-5213-433b-8fbf-1945bb4496e1",
+    "rootTask": {
+        "id": "9825d5e8-5213-433b-8fbf-1945bb4496e1",
+        "name": "",
+        "resourceUri": "/data-services/v1beta1/async-operations/9825d5e8-5213-433b-8fbf-1945bb4496e1",
+        "type": "task"
+    },
+    "services": [
+        "private-cloud-business-edition"
+    ],
+    "sourceResource": {
+        "name": "VVOL-ds2",
+        "resourceUri": "/api/v1/datastores/fdcc724f-3f22-5fa6-8477-81f0fd701512",
+        "type": "Datastore"
+    },
+    "startedAt": "2024-03-29T00:56:16.981298562Z",
+    "state": "SUCCEEDED",
+    "subtreeTaskCount": 2,
+    "suggestedPollingIntervalSeconds": 30,
+    "type": "task",
+    "updatedAt": "2024-03-29T00:59:23.013384523Z",
+    "userId": "ronald.dharma@hpe.com"
+}
+```
+
+*T﻿he above figure shows the termination of machine-instance in AWS reached 100% progress and completed successfully.*
+
+## Summary
+
+This blog post introduces you to the new set of HPE Greenlake API for data services named virtualization to support resources from a hypervisor that includes VMware (6.7.X, 7.X and 8.X) for on-premises, AWS EC2 and Azure for cloud-public-provider. This API set will evolve throughout next release in the future. The March 2024 announcement introduces V1Beta1 of the API version. This set of APIs is documented at HPE GreenLake Developer [website ](https://developer.greenlake.hpe.com) and it is available for download as Open API 3.1.X specification in JSON file.
```

Comment:
HPE GreenLake with "a capital "L"

Maybe I would slightly change the first sentence of the Summary

This blog post introduces you to the new set of HPE Greenlake API for data services named virtualization to support resources from a hypervisor that includes VMware (6.7.X, 7.X and 8.X) for on-premises, AWS EC2 and Azure for cloud-public-provider. 

into:

This blog post introduces you the HPE GreenLake for Virtualization API that is part of the data services family of APIs for HPE GreenLake. This set of Virtualization APIs supports resources from a hypervisor that includes VMware (6.7.X, 7.X and 8.X) for on-premises, AWS EC2 and Microsoft Azure for cloud-public-provider. 


--> Add Microsoft in front of Azure.